packages/webdriverio/src/commands/browser/switchWindow.ts

```diff
@@ -45,7 +45,7 @@ export async function switchWindow (
         throw new Error('Unsupported parameter for switchWindow, required is "string" or a RegExp')
     }
 
-    const currentWindow = await this.getWindowHandle()
+    const currentWindow = await this.getWindowHandle().catch(() => null)
```

Comment:
In what cases do we expect the command not to be available?

Reply:
Maybe, when there is no active window or the context is not initialized properly. Although, I think BiDi is somehow handling these cases and thus we are seeing these issues in classic mode only.